CMakeLists.txt

```diff
@@ -64,13 +71,28 @@ else()
         set(LAR_CONTENT_LIBRARY_NAME "LArContent")
     endif()
     project(${LAR_CONTENT_LIBRARY_NAME})
+    
+    # DL Content
+    option(PANDORA_LIBTORCH "Flag for building against LibTorch" OFF)
+    if(PANDORA_LIBTORCH)
+        set(DL_PROJECT_NAME "LArDLContent")
+        set(LAR_DL_CONTENT_LIBRARY_NAME "${DL_PROJECT_NAME}")
+    endif()
 
     # project version
     # ATTN This package supports two build systems; please ensure version is specified here *and* in ups/product_deps
-    set(${PROJECT_NAME}_VERSION_MAJOR 03)
-    set(${PROJECT_NAME}_VERSION_MINOR 19)
-    set(${PROJECT_NAME}_VERSION_PATCH 03)
-    set(${PROJECT_NAME}_VERSION "${${PROJECT_NAME}_VERSION_MAJOR}.${${PROJECT_NAME}_VERSION_MINOR}.${${PROJECT_NAME}_VERSION_PATCH}")
+    foreach(PROJ IN LISTS PROJECT_NAME DL_PROJECT_NAME)
+        set(${PROJ}_VERSION_MAJOR 03)
+        set(${PROJ}_VERSION_MINOR 19)
```

Comment:
How does adding the dependency, in this rather soft way, affect the version? Would you agree that [semver](https://semver.org) would suggest this is v03_20_00?

Reply:
I agree this is a minor version update, not a patch. I'm currently operating on the assumption this will be v03_21_00, with the track/shower BDT update being v03_20_00 ahead of this - I've held off making the version change until we're sure about the ordering of the releases.